tsconfig.json

```diff
@@ -3,12 +3,18 @@
     "lib": ["es2015"],
     "target": "es5",
     "module": "commonjs",
-    "declaration": true,
+    "declaration": false,
     "outDir": "./dist",
+    "allowJs": true,
```

Comment:
Why the change?

Reply:
this allows us to have .js files as a part of module code along with .ts